src/modules/footer/components/attributionInfo/AttributionInfo.js

```diff
@@ -22,15 +22,29 @@ export class AttributionInfo extends BaElement {
      */
 	createView() {
 		const translate = (key) => this._translationService.translate(key);
-		const { active } = this._state;
+		const { active, zoom } = this._state;
 
 		if (typeof active !== undefined && active.length > 0) {
+
 			const geoResource = this._georesourceService.byId(active[0].id);
+
 			if (!geoResource) {
 				return nothing;
 			} 
-			// Not yet implemented
-			geoResource.attribution ? this._content = geoResource.attribution : this._content = translate('map_attributionInfo_fallback');
+
+			const attributions = geoResource.getAttribution(zoom);
+
+			if (!attributions) {
+				return nothing;
+			} 
```

Comment:
Have a look @ attribution typedef in `geoResources.js`.
An attribution object MAY also contain an `url` property. In this case, we need an anchor here.